From 13342b50a2def0dbb346de9505b1a9a24b753d5b Mon Sep 17 00:00:00 2001
From: Dagobert Michelsen <dam@opencsw.org>
Date: Sat, 15 Nov 2014 10:53:34 +0100
Subject: [PATCH] Add /opt/csw/{gnu,bin} to PATH in a2x.py

This allows us to leverage the gnu tools which is what the app was
written against.  It prevents blowups when non-gnu tools get gnu
options passed.
---
 a2x.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/a2x.py b/a2x.py
index c30e15b..4221742 100755
--- a/a2x.py
+++ b/a2x.py
@@ -66,6 +66,8 @@ BACKEND_OPTS = ''
 
 OPTIONS = None  # These functions read verbose and dry_run command options.
 
+os.environ["PATH"] = "/opt/csw/gnu:/opt/csw/bin:" + os.environ["PATH"]
+
 def errmsg(msg):
     sys.stderr.write('%s: %s\n' % (PROG,msg))
 
-- 
2.0.3

